pkg/sql/schemachanger/testdata/end_to_end/create_sequence/create_sequence.side_effects

```diff
@@ -29,21 +29,13 @@ upsert descriptor #104
   +      family: IntFamily
   +      oid: 20
   +      width: 64
-  +  - defaultExpr: unique_rowid()
```

Comment:
ah, good reminder that we should try to review what's in these expectation files closely